lib/cypress/integration/index.spec.js

```diff
@@ -1,3 +1,5 @@
+import { describe } from "mocha";
```

Comment:
where is `describe` being used? 🤔

Reply:
@manuelschroederdev ... your plugin is trying to be too funny

Reply:
@manuelschroederdev I just checked, it's not even a plugin that's responsible for this. 👀